predicators/spot_utils/README.md

```diff
@@ -1,5 +1,41 @@
 # Spot Utils
 
+## How to run your own Spot environment
+
+> Last Updated: 04/11/2024
+
+**Steps:**
+- Set up the codebase, perception pipeline, and Spot
+  - You need to have access to a GPU server for the perception pipeline (e.g., the Detic-SAM pipeline)
+  - You need to connect to Spot (through WiFi or ethernet cable). Spot in its own WiFi AP mode is on IP `192.168.80.3`.
```

Comment:
This is only true for the LIS spot; so maybe say "Spot in its own WiFi AP mode will have IP address `192.168.80.3`

Reply:
done!